src/benchmarks/pmembench.cpp

```diff
@@ -389,7 +436,9 @@ pmembench_print_header(struct pmembench *pb, struct benchmark *bench,
 	       "latency-avg[nsec];"
 	       "latency-min[nsec];"
 	       "latency-max[nsec];"
-	       "latency-std-dev[nsec]");
+	       "latency-std-dev[nsec];"
```

Comment:
that's an interface change, be sure to notify everyone who might depend on this (validation)